### PR TITLE
For testing purpose- DO NOT MERGE THIS PR

### PR DESCRIPTION
**Pull request by olusegz07**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EXUI-2344

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by MunishSharmaHMCTS — DISMISSED** [review]

**Review by Josh-HMCTS — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "7.0.71",
+  "version": "7.0.73",
```
Comment (Josh-HMCTS) [inline package.json:3]: Please use pre-release version number if creating pre-release version

**Review by andywilkinshmcts — CHANGES_REQUESTED** [review]
I think an error has been made in first-error.pipe.ts. I can't see a reason for changing the text "is required" to "Save and continue"

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit-page/case-edit-page.component.ts [hunk 2]
```diff
@@ -214,7 +214,7 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked, OnDestro
               this.addressService.setMandatoryError(true);
               this.caseEditDataService.addFormValidationError({ id: `${path}_${path}`, message: `An address is required` });
             } else {
-              this.caseEditDataService.addFormValidationError({ id, message: `%FIELDLABEL% is required`, label });
+              this.caseEditDataService.addFormValidationError({ id, message: `%FIELDLABEL% Save and continue`, label });
```
Comment (andywilkinshmcts) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit-page/case-edit-page.component.ts:217]: This seems to be an error - I don't see why we would want this change

**Review by andywilkinshmcts — CHANGES_REQUESTED** [review]

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/utils/first-error.pipe.spec.ts [hunk 3]
```diff
@@ -65,7 +65,7 @@ describe('FirstErrorPipe', () => {
     expect(message).toBe(ERROR_MESSAGE);
   });
 
-  it('should return exact error along with label name when field value is MANDATORY', () => {
+  xit('should return exact error along with label name when field value is MANDATORY', () => {
```
Comment (andywilkinshmcts) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/utils/first-error.pipe.spec.ts:68]: Why is this disabled?

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/utils/first-error.pipe.ts [hunk 4]
```diff
@@ -35,7 +35,7 @@ export class FirstErrorPipe implements PipeTransform, OnDestroy {
 
     let errorMessage: string;
     if (keys[0] === 'required') {
-      errorMessage = '%FIELDLABEL% is required';
+      errorMessage = '%FIELDLABEL% Save and continue';
```
Comment (andywilkinshmcts) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/utils/first-error.pipe.ts:38]: Why is this change required?

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit-page/case-edit-page.component.ts (hunk 2 above)
Comment (olusegz07) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit-page/case-edit-page.component.ts:217]: This text was used for test purposes only to demonstrate the Welsh translation enhancement works. The phrase "is required" is not currently in the Welsh database. As stated above, the PR is not to be merged.

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/utils/first-error.pipe.ts (hunk 4 above)
Comment (olusegz07) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/utils/first-error.pipe.ts:38]: This text was used for test purposes only to demonstrate the Welsh translation enhancement works. The phrase "is required" is not currently in the Welsh database. As stated above, the PR is not to be merged.
